code/modules/mob/living/carbon/human/human_abilities.dm

```diff
@@ -116,6 +116,18 @@
 /datum/action/human_action/smartpack/repair_form/cooldown_check(var/obj/item/storage/backpack/marine/smartpack/S)
 	return S.repairing
 
+/datum/action/human_action/activable/rto_pack/use_phone
+	name = "Use Phone"
+	action_icon_state = "phone"
```

Comment:
put this in /New() like how this does it
`/datum/action/item_action/specialist/fire_shield/New(var/mob/living/user, var/obj/item/holder)
	..()
	name = "Activate Fire Shield"
	button.name = name
	button.overlays.Cut()
	var/image/IMG = image('icons/obj/items/clothing/cm_suits.dmi', button, "pyro_armor")
	button.overlays += IMG`

also you should probably change your phone to just the phone itself (not the background) and then apply it via overlay onto the button